Dockerfile

```
@@ -2,12 +2,15 @@
 
 # This enforces that the packages downloaded from the repositories are the same
 # for the defined date, no matter when the image is built.
-ARG NOBLE_DATE=20240801
-ARG APT_UPDATE_SNAPSHOT=${NOBLE_DATE}T030400Z
+ARG UBUNTU_TAG=noble-20240827.1
+ARG APT_UPDATE_SNAPSHOT=20240827T030400Z
+
+# Built-in binaries version
+ARG BUILTINS_VERSION=0.5.0
```

```suggestion
ARG BUILTINS_VERSION=0.6.0
```
Can we bump already?